test/integration/fire_cloud_client_test.rb

```diff
@@ -290,8 +290,7 @@ def test_get_configurations
   def test_create_configuration_template
     puts "#{File.basename(__FILE__)}: '#{self.method_name}'"
 
-    workflow_methods = @fire_cloud_client.get_methods(entityType: 'workflow')
-    method = workflow_methods.sample
+    method = @fire_cloud_client.get_method('single-cell-portal', 'split-cluster', 1)
```

Comment:
Out of curiosity, how does this address the flaky test?

Reply:
Because that is a method/configuration that is part of portal source that is run in another integration test, so we know that it will always exist and always be available.  Before, we just queried for a method and _hoped_ that it had available configurations.  Most do, but not all, so that is why occasionally this test would fail at the next assertion.